recipes/arb-bio/6.1.rev16223/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "ARB" %}
+{% set version = "6.1.rev16223" %}
```

Comment:
The next release version is likely to follow the 6.1.0-rc1 and 6.1.0 scheme. That should be OK, as in conda strings are "less than" integers unless the string is "post". 

```
>>> from conda.version import VersionOrder
>>> VersionOrder("6.1.rev16223") < VersionOrder("6.1")
True
>>> VersionOrder("6.1.rev16223") < VersionOrder("6.1.0")
True
```

What I considered was naming it "6.0.5+6.1.rev16223" (after `+` is the local version, which is always less significant than the main version) so that the official release version would still be preferred unless someone requested the build specifically. OTOH, naive users are probably better served with the current version that has 3 years more development in it.